### PR TITLE
fix: Adding error messages for templates and disruptions not found

### PR DESCRIPTION
**Pull request by alainajaffer**
View a template or disruption, delete it press back arrow on the browser.

Chris has stated he wants both to navigate back to the dashboard. Design of page has been confirmed with him too.

### CODE REVIEW

File: site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx [hunk 1]
```diff
@@ -240,7 +241,10 @@ export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props:
     );
 
     if (!disruption) {
-        throw new Error("No disruption found for network consequence page");
+        if (ctx.res) {
```
Comment (alvak1) [inline site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx:244]: Need to add same block for `type-of-consequence` page

File: site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx [hunk 2]
```diff
@@ -240,7 +241,10 @@ export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props:
...
+            redirectTo(ctx.res, `${DISRUPTION_NOT_FOUND_ERROR_PAGE}${!!ctx.query?.template ? "?template=true" : ""}`);
```
Comment (alvak1) [inline site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx:245]: Is template query param required?

Comment (alainajaffer) [reply site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx:245]: yes to know what text to show

File: site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx (hunk 1 above)
Comment (alvak1) [reply site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx:244]: Also required for `create-disruption` page

Comment (alainajaffer) [reply site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx:244]: done

Comment (alainajaffer) [reply site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx:244]: not needed on create disruptions it reacts differently

**Review by alvak1 — APPROVED** [review]
Approve

**Pull request merged**